Header header

# скорость вперед
float32 forward

# скорость вправо
float32 right

# скорость вниз
float32 down

# скорость по глубине
float32 depth

# скорость на север
float32 north

# скорость на восток
float32 east